dist/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8">
   <title>Skeleton Framework Test File</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+  <link href="http://fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
```

Comment:
I added http because the live-serve doesn't work on my PC. So I load it with file:// and the font cannot load over file:// .